src/lib/stitching/gravity/v2/stitching.ts

```diff
@@ -410,202 +673,209 @@ export const gravityStitchingEnvironment = (
           },
         },
       },
-      User: {
-        devices: {
+      Partner: {
+        searchCriteriaConnection: {
           fragment: gql`
-            ... on User {
+          ... on Partner {
             internalID
           }
-          `,
-          resolve: ({ internalID: userId }, _args, context, info) => {
+        `,
+          resolve: ({ internalID: partnerID }, args, context, info) => {
             return info.mergeInfo.delegateToSchema({
               schema: gravitySchema,
               operation: "query",
-              fieldName: "_unused_gravity_devices",
-              args: { userId },
+              fieldName: "_unused_gravity_searchCriteriaConnection",
+              args: {
+                partnerID,
+                ...args,
+              },
```

Comment:
Here we're actually doing the wiring to get the data all connected together. 

In short, how this works is: 
- We're looking at the parent field `partner` to see what its id is
- Once we have that, we forward it on to the native `searchCriteriaConnection` `partnerID` argument. 
- The `_unused_gravity_searchCriteriaConnection` is for remapping purposes. By default we add `_unused_gravity_` to the field name, and then we create our own `searchCriteriaConnection`, where we want (eg on `Partner`, and later on `Me`)

Reply:
Thank you for the details on this!! The `_unused_gravity` thing was a bit unclear to me, now I get it! ❤️